### PR TITLE
[BLOGS-1422] Set blogs ATI content labels to unlock UAS on comments

### PR DESCRIPTION
**Pull request by benolot**
Ticket: https://jira.dev.bbc.co.uk/browse/BLOGS-1422

### CODE REVIEW

File: src/Controller/Helpers/Services/AtiAnalyticsHelper.php [hunk 1]
```diff
@@ -25,11 +25,13 @@ public function setChapterOneVariable(string $chapterOne): void
         $this->chapterOne = $chapterOne;
     }
 
-    public function makeLabels(string $chapterOne, ?Blog $blog = null, bool $hasVideo = false): AtiAnalyticsLabels
+    public function makeLabels(string $chapterOne, string $contentType, ?Blog $blog = null, bool $hasVideo = false,?string $guid = ""): AtiAnalyticsLabels
     {
         $labels = [
             'destination' => $this->getDestination(),
             'section' => $chapterOne,
+            'contentId' => 'urn:bbc:isite' . ($guid ? ":" . $guid : ''),
```
Comment (BjornTwachtmann) [inline src/Controller/Helpers/Services/AtiAnalyticsHelper.php]: Should we be setting this field if we don't have a GUID? 'urn:bbc:isite' seems weird

Comment (benolot) [reply src/Controller/Helpers/Services/AtiAnalyticsHelper.php]: That's the standard yeah. (See programmes pages without pids that go "urn:bbc:pips")

File: src/Controller/Helpers/Services/AtiAnalyticsHelper.php [hunk 2]
```diff
@@ -25,11 +25,13 @@ public function setChapterOneVariable(string $chapterOne): void
         $this->chapterOne = $chapterOne;
     }
 
-    public function makeLabels(string $chapterOne, ?Blog $blog = null, bool $hasVideo = false): AtiAnalyticsLabels
+    public function makeLabels(string $chapterOne, string $contentType, ?Blog $blog = null, bool $hasVideo = false,?string $guid = ""): AtiAnalyticsLabels
```
Comment (Tasiobg) [inline src/Controller/Helpers/Services/AtiAnalyticsHelper.php]:
```suggestion
    public function makeLabels(string $chapterOne, string $contentType, ?Blog $blog = null, bool $hasVideo = false, ?string $guid = ""): AtiAnalyticsLabels
```

Comment (Tasiobg) [reply src/Controller/Helpers/Services/AtiAnalyticsHelper.php]: a part from this missing space, looks good to me

**Comment by benolot** [comment]
Test this please

**Pull request merged**